.compilerc

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Yep! See https://github.com/electron/electron-compile#how-do-i-set-up-babel--less--whatever-the-way-i-want